performant-software/basira
# BASIRA #132 - Use ItemCollection in Artworks index

### What this PR does

- Per #132:
  - Uses the `ItemList` component to display artworks on main Artworks index

### Status


### Steps to test (when deployed)

1. Visit https://basiraproject-staging.herokuapp.com/
2. Log in
3. Verify that the list of artworks immediately appears as in the screenshots in #132 (with the exception that the grid is 3 items across instead of 4)
